TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/testers/RazArmTest.java

```diff
@@ -0,0 +1,25 @@
+package org.firstinspires.ftc.teamcode.opmodes.testers;
+
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+
+import org.firstinspires.ftc.teamcode.Bindings;
+import org.firstinspires.ftc.teamcode.Robot;
+import org.firstinspires.ftc.teamcode.opmodes.DefaultRaz;
+
+@TeleOp(name = "Raz Arm Test")
+public class RazArmTest extends DefaultRaz {
+    @Override
+    public void initialize() {
+
+    }
+
+    @Override
+    public void execute() {
+        Robot.getInstance().ARM.telemetry(telemetry);
```

Comment:
Don't you need to add telemetry.update() ?

Reply:
no the op mode do it alone (the thing DefaultRaz extends from)